src/core/engine/engine_dispatcher.inl

```diff
@@ -246,6 +246,17 @@ namespace otto::core::engine {
         ctx.stroke(i.color, i.line_width);
       };
     }
+    if (engine_name == "PingPong") {
```

Comment:
this function is an abomination. We need a better approach. Probably a more general EngineRegistry.

Reply:
I completely agree. If gh had a vomiting emoji, I would have used it.

Reply:
:vomiting_face: